app/javascript/components/widgets/numbers/carousel.jsx

```diff
@@ -32,19 +33,24 @@ const getEventCounts = gql`
   }
 `;
 
+const STAT_FORMAT = '0,0';
+
 class SubscribedEvents extends React.Component {
   componentDidMount() {
     this.props.subscribeToPublishedEvents();
   }
 
   render() {
     const { githubPull, githubCommit, slackMessage } = this.props;
+    const githubPullCount = numeral(githubPull).format(STAT_FORMAT);
+    const githubCommitCount = numeral(githubCommit).format(STAT_FORMAT);
+    const slackMessageCount = numeral(slackMessage).format(STAT_FORMAT);
```

Comment:
what is the reason for using this numeral library? could we accomplish this without adding the dependency?

Reply:
🤷‍♂ probably but that's more code we'd have to maintain, I'd rather let numeral do the formatting personally

Reply:
could we use this? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString#Using_toLocaleString

Reply:
I'm ok with 3.8kB added for an expressive way to format numbers
https://bundlephobia.com/result?p=numeral@2.0.6